shared/constants/chat2/message.js

```diff
@@ -728,48 +762,61 @@ export const makePendingTextMessage = (
 export const makePendingAttachmentMessages = (
```

Comment:
Ok, yeah I just didn't want to change it unless since I thought maybe people liked it being an array thing. I can make it just a single guy (since that is how it is always used anyway).